### DISCUSSION TITLE
Update issue service not starting

### DISCUSSION THREAD

**Discussion by rbcrack** [Security-Onion-Solutions/securityonion#8862] [Unsupported Versions] [answered]
Hi,

I have ran soup a few times and did all the updates successfully and after I rebooted the machine now all the services are showing wait_start.

I have docker service running just fine.

I have waited overnight for the services to start but no luck.

Any help will be appricated

![image](https://user-images.githubusercontent.com/18475926/194111533-fb0b97af-5096-49ff-b4b9-823a7c384531.png)

**Comment by reyesj2** [comment, marked as answer]
I have had success with a forceful restart of all containers. You can try this with

```
sudo docker restart $(docker ps -qa)
```
I would follow up with reviewing your /root/soup.log for any issues you may have missed.

**Reply by rbcrack** [reply]
I have reinstalled the server please close the case

**Reply by rbcrack** [reply]
Thank You everyone for  your help
